### PR TITLE
cleanup, add initial chat view

### PR DESCRIPTION
**Pull request by uoftm**

### CODE REVIEW

File: src/view/SignupView.java [hunk 1]
```diff
@@ -25,13 +24,25 @@ public class SignupView extends JPanel implements ActionListener, PropertyChange
...
 
   public SignupView(SignupController controller, SignupViewModel signupViewModel) {
+    this.setBackground(Color.YELLOW);
 
     this.signupController = controller;
-    this.signupViewModel = signupViewModel;
     signupViewModel.addPropertyChangeListener(this);
 
+    JPanel body = new JPanel();
+    body.setBackground(Color.RED);
```
Comment (uoftm) [inline src/view/SignupView.java:33]: This colour is great, although a future PR should probably pull it out into a separate file of UI constants

Comment (alexrosen45) [reply src/view/SignupView.java:33]: Can you create a new issue for this so we can track it? Otherwise, looks great

**Review by alexrosen45 — APPROVED** [review]
We will need to combine the code in `Main` and `Main2` soon, or perhaps split them up in a smarter way. But, everything else looks great!

File: src/view/SignupView.java (hunk 1 above)
Comment (uoftm) [reply src/view/SignupView.java:34]: Added a commit for a TODO, maybe that's enough? (If not I can also create the issue)

**Comment by uoftm** [comment]
Created https://github.com/uoftm/csc207project/issues/5 for getting rid of Main2

File: src/view/SignupView.java (hunk 1 above)
Comment (alexrosen45) [reply src/view/SignupView.java:34]: Yup, that's fine. You're good to merge.

**Pull request merged**